GUI/utils/header.py

```diff
@@ -84,20 +85,49 @@ def __init__(self, w=1000, h=500, *args, **kwargs):
         hbox.addWidget(splitter3)
         self.setStyleSheet(qstr)
 
-        self.pBar= QProgressBar(self)
-        #self.pBar.setGeometry(0, 0, 10, 30)
-        #self.pBar.resize(10,10)
-        self.pBar.setFixedWidth(200)
+        
         splitter4= QSplitter(Qt.Horizontal)
-        splitter4.addWidget(self.pBar)
-        #splitter4.setSizes([self.height / 30, self.height / 30])
+
+        #self.pBar= QProgressBar(self)                                  PROGRESS BAR
+        self.pBar = progressBar.ProgressBar()
+        #pBar.setGeometry(0, 0, 10, 30)
+        #pBar.setFixedWidth(250)
+        #pBar.show()
+        splitter4.addWidget(self.pBar.label)
+        splitter4.addWidget(self.pBar.pBar)
```

Comment:
Can you modify the attributes so they are more descriptive? `self.pBar.pBar` is a bit confusing

Reply:
I'm not sure what to change it to because its just the progress bar itself.

Reply:
One option is to make it `self.pBarContainer.pBar`. The exact name doesn't matter so much, the main thing is that they should be different. Ideally, the name reflects the pattern that `self.pbar` is the class/container for the `PyQt5` progress bar component.